lib/naughty/core.lua.in

```diff
@@ -237,6 +237,77 @@ function naughty.getById(id)
     end
 end
 
+--- Install expiration timer for notification object
+local function set_up_timer(notification, timeout)
+    local die = function (reason)
+        naughty.destroy(notification, reason)
+    end
+    if timeout > 0 then
+        local timer_die = timer { timeout = timeout }
+        timer_die:connect_signal("timeout", function() die(naughty.notificationClosedReason.expired) end)
+        if not suspended then
+            timer_die:start()
+        end
+        notification.timer = timer_die
+    end
+    notification.die = die
+end
+
+--- Set new notification timeout
+-- @param notification Notification object, which timer is to be reset
+-- @param new_timeout Time in seconds after which notification disappears
```

Comment:
Please use punctuation at the end, and `@tparam` to specify the type for the args.